app/models/user.rb

```diff
@@ -55,6 +56,9 @@ class User < ApplicationRecord
     unpwn: true,
     allow_nil: true,
     unless: :skip_password_validation?
+
+  validates :full_name, length: { within: 0..20 }, allow_nil: true
```

Comment:
Is there any reason to limit name to 20 characters?

Reply:
At first, was because of the space and the current design, that the overflow with spaces takes more than one line, not like the display_id. So that could break the layout like this:

![image](https://user-images.githubusercontent.com/8846835/167505925-84e8f8b9-dfa0-40c2-bae5-073f7201c582.png)

I added CSS to get this(can you check it?):

![image](https://user-images.githubusercontent.com/8846835/167506037-c1588a5c-2f07-4894-b462-51240fe07fcf.png)


So should I put the limit to 40 like the username?

Reply:
I think we should not really limit the name. It would be ok to keep it 255 (default DB varchar size if I remember well).

Reply:
I updated the limit :+1: